--- origsrc/libinfinity-0.6.6/infinoted/Makefile.am	2015-02-24 19:50:00.551717300 -0600
+++ src/libinfinity-0.6.6/infinoted/Makefile.am	2015-06-01 17:53:13.537397500 -0500
@@ -25,6 +25,7 @@ libinfinoted_plugin_manager_0_6_la_LDFLA
 
 libinfinoted_plugin_manager_0_6_la_LIBADD = \
 	${top_builddir}/libinfinity/libinfinity-$(LIBINFINITY_API_VERSION).la \
+	$(libdaemon_LIBS) \
 	$(infinoted_LIBS) \
 	$(infinity_LIBS)
 
